src/main/java/org/terasology/projsndwv/components/BeeComponent.java

```diff
@@ -17,5 +17,14 @@
 
 import org.terasology.entitySystem.Component;
 
-public class BeeComponent implements Component {
+public final class BeeComponent implements Component {
+    // TODO: Make enum
+    /**
+     * Bee type, that is, whether a bee is a drone, princess, or queen.<br/>
+     * <br/>
+     * 0 - Drone<br/>
+     * 1 - Princess<br/>
+     * 2 - Queen
+     */
+    public int type;
```

Comment:
I would prefer to see this as an enum for clarity.

Reply:
@jellysnake Mind directing me to an example of an enum being used in prefabs? I don't know how prefabs initialize enum fields, how the constructor must be formatted, etc.